docs/user-guide/lazy/execution.md

```diff
@@ -77,3 +77,18 @@ shape: (27, 6)
 │ 77768 ┆ TINHEADNED                ┆ 1139665457  ┆ 1536497404 ┆ 4434          ┆ 103        │
 └───────┴───────────────────────────┴─────────────┴────────────┴───────────────┴────────────┘
 ```
+
+### Cancellable Executions
+
+With background mode, you can run a query in the background and get a handler for
+an the in process query. The handler can be used to to do something else while
+the query is running or be canceled.
+
+Here we run a background query and cancel the query if it took more than
+`max_wait_time` seconds to complete.
+
+!!! warning "background mode is **unstable**"
+
+    Background mode may be changed at any point as a breaking change.
```

Comment:
`as a non-breaking change`

Reply:
I can see how the phasing can be unclear. I updated this PR's wording to be the same as the docstring:

https://github.com/pola-rs/polars/blob/ca6d46cb6746f8da7a71d02c9c00a7e0679a4d02/py-polars/polars/lazyframe/frame.py#L1882-L1884